apps/projection/src/app/ui/card/card.component.html

```diff
@@ -1,29 +1,22 @@
-<div class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3" [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px"
-  />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px"
-  />
+<div
+  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
+  [class]="customClass">
+  <ng-content select="[student]"></ng-content>
+  <ng-content select="[teacher]"></ng-content>
+  <ng-content select="[city]"></ng-content>
```

Comment:
the goal is to make a highly customizable component. That's the goal solution but why do you need to create one for each card ? 
If you need to add a new card, you will need to come here a add a new ng-content. Just create a global selector and this will perfect